misc/vim/syntax/snakemake.vim

```diff
@@ -49,7 +49,7 @@ source $VIMRUNTIME/syntax/python.vim
 syn keyword pythonStatement	include workdir onsuccess onerror
 syn keyword pythonStatement	ruleorder localrules configfile group
 syn keyword pythonStatement	touch protected temp wrapper conda shadow
-syn keyword pythonStatement	input output params message threads resources singularity wildcard_constraints
+syn keyword pythonStatement	input output params wildcards message threads resources singularity wildcard_constraints
 syn keyword pythonStatement	version run shell benchmark snakefile log script
 syn keyword pythonStatement	rule subworkflow nextgroup=pythonFunction skipwhite
```

Comment:
Also add in:
- [ ] priority
- [ ] directory
- [ ] ancient
- [ ] onstart
- [ ] pipe
- [ ] checkpoint (also in lines 58 and 61)

Maybe consider functions:
- [ ] unpack
- [ ] expand

And consider objects:
- [ ] config
- [ ] checkpoints
- [ ] rules

Update line 24 to current version

Reply:
- [ ] directory
- [ ] ancient
- [x] onstart
- [ ] pipe
- [x] checkpoint (also in lines 58 and 61)

Maybe consider functions:

- [ ] unpack
- [ ] expand

And consider objects:

- [x] config
- [x] checkpoints
- [x] rules